test/unit/model/jdk-install-test.js

```diff
@@ -187,51 +182,46 @@ describe('JDK installer', function() {
             jdk.getMsiSearchScriptLocation(), jdk.getMsiSearchScriptData());
           expect(Util.executeFile).to.have.been.calledWith(
             'powershell', jdk.getMsiSearchScriptPowershellArgs(jdk.getMsiSearchScriptLocation()));
-          done();
         });
       });
     });
 
     describe('on macos', function() {
-      it('should not select jdk for installation if no java detected', function(done) {
+      it('should not select jdk for installation if no java detected', function() {
         sandbox.stub(Platform, 'getOS').returns('darwin');
         let jdk = new JdkInstall(installerDataSvc, 'url', 'jdk8.msi', 'jdk8', 'sha');
         mockDetectedJvm('');
         return jdk.detectExistingInstall(function() {
```

Comment:
tests should be in then() callback. Now it works because callback is never called.